src/main/clm_driver.F90

```diff
@@ -111,8 +111,9 @@ subroutine clm_drv(doalb, nextsw_cday, declinp1, declin, rstwr, nlend, rdate, ro
     !
     ! !USES:
     use clm_time_manager     , only : get_curr_date
-    use clm_varctl           , only : use_lai_streams
+    use clm_varctl           , only : use_lai_streams, fates_spitfire_mode
     use SatellitePhenologyMod, only : lai_advance
+    use CNFireFactoryMod     , only : scalar_lightning
```

Comment:
Should be renamed to a FATES only name.

Reply:
update per 08b1f8781a3dd3442e52e4719b840ae2559498b9